renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -45,6 +45,34 @@ static std::string ToHumanStr(const VkAttachmentStoreOp &el)
   END_ENUM_STRINGISE();
 }
 
+void WrappedVulkan::AddImplicitResolveResourceUsage(uint32_t subpass)
+{
+  ResourceId rp = m_BakedCmdBufferInfo[m_LastCmdBufferID].state.renderPass;
+  ResourceId fb = m_BakedCmdBufferInfo[m_LastCmdBufferID].state.framebuffer;
+
+  VulkanCreationInfo::Framebuffer fbinfo = m_CreationInfo.m_Framebuffer[fb];
+  VulkanCreationInfo::RenderPass rpinfo = m_CreationInfo.m_RenderPass[rp];
+
+  // Ending a render pass instance performs any multisample operations
+  // on the final subpass. ~0U is the end of a RenderPass.
+  if(subpass == ~0U)
+    subpass = (uint32_t)rpinfo.subpasses.size() - 1;
+  else
+    subpass = m_BakedCmdBufferInfo[m_LastCmdBufferID].state.subpass;
+
+  std::vector<ResourceId> fbattachments = m_BakedCmdBufferInfo[m_LastCmdBufferID].state.fbattachments;
```

Comment:
Less important because it's only a small vector, but it's still better to avoid copies where it's not necessary for things larger than a register.